DoSomethingArchive/legacy-website
# Deprecates static homepage.  Conditionally restricts finder

#### What's this PR do?
- Removes the old static homepage markup and conditional logic
- Removes associated stylesheets and references
- Conditionally restricts the campaign finder display via the show_campaign_finder theme setting
- Sets template variables for campaign finder & sponsor toggles to remove db calls from the tpl.
#### Where should the reviewer start?

The most significant changes appear here: https://github.com/desmondmorris/dosomething/blob/369157275c10b887fd43b5dc485d24df2e01508f/lib/themes/dosomething/paraneue_dosomething/templates/home/node--home.tpl.php
#### How should this be manually tested?
- Pull down my branch and clear cache.
- Visit theme admin page for paraneue_dosomething.  Toggle the campaign finder & sponsors flags and check the homepage.
#### Any background context you want to provide?

Since moving to a homepage node, there is no real reason to continue lugging around the static homepage markup.  It has been necessary to date to keep the international platform happy.

This work is necessary to grant privileged users on international affiliate sites the ability to remove the campaign finder from the homepage.
#### What are the relevant tickets?

Fixes #2224

cc: @DFurnes @weerd - please double check this and test in your env if you get a chance.
